tests/test_celebrity.py

```diff
@@ -18,18 +18,19 @@
 GOOD_DATES = [
     datetime.date(2020, 9, 18),
     datetime.date(2019, 9, 18),
-    datetime.date(2016, 9, 18)
+    datetime.date(2016, 9, 18),
 ]
 
 
 @pytest.fixture
 def datetime_mock(monkeypatch):
-    class MyDateTime:
-        @classmethod
-        def today(cls):
+    class MockDateTime:
+
```

Comment:
please remove blank line - line 28

Reply:
PyCharm auto-formatter does not want to do it. Also, according to [PEP 8](https://www.python.org/dev/peps/pep-0008/#blank-lines) 

> Method definitions inside a class are surrounded by a single blank line

So it would seem that PyCharm is correct here.